docs/cloud-soar/compared-to-automation-service.md

```diff
@@ -31,23 +31,23 @@ Cloud SOAR also offers many more features than the Automation Service, including
 
 | Feature | Automation<br/>Service | Cloud SOAR |
 | :-- | :-- | :-- |
-| Advanced automation capabilities <br/>- Daemons <br/>- Triggers | | ![check](/img/reuse/check.png) |
```

Comment:
@jpipkin1 the table looks like it was in alphabetical order, why was this restructured?

Reply:
@JV0812, I got a request to move all the Automation Service capabilities to the top.